File: C Programs/Week1/pascal.c
#include <stdio.h>
void pascal_triangle(int n) {
    for (int i = 0; i < n; i++) {
        int val = 1;
        for (int k = 0; k < n - i; k++) printf(" ");
        for (int j = 0; j <= i; j++) {
            printf("%d ", val);
            val = val * (i - j) / (j + 1);
        }
        printf("\n");
    }
}
int main() {
    int rows;
    printf("Enter number of rows: ");
    scanf("%d", &rows);
    pascal_triangle(rows);
    return 0;
}

File: Python Programs/Week1/pascal.py
def pascal(n):
    for i in range(n):
        v = 1
        print(" " * (n - i), end="")  
        for j in range(i + 1):
            print(f"{int(v)} ", end="")
            v = v * (i - j) / (j + 1)
        print()
n = int(input("Enter number of rows for Pascal's Triangle: "))
pascal(n)
